cli/cmd/install.go

```diff
@@ -220,17 +222,37 @@ func render(config installConfig, w io.Writer, options *installOptions) error {
 	// Render raw values and create chart config
 	rawValues, err := yaml.Marshal(config)
 	if err != nil {
-		return err
+		log.Fatalf("Could not marshal `config` into JSON: %s", err)
```

Comment:
I don't know the answer to this right now. It just seemed like it was the proper way to fail based off reading through other packages. I can investigate and get back to you!

Reply:
In the `cli/` directory, we should always be returning an error instead of using `log.Fatal. Returning an error to cobra will cause it to print the help text for the command that was run, whereas calling `log.Fatal` will exit immediately without any help text. If you look at the other files in this directory, none of them are using `log.Fatal`.

We do use `log.Fatal` in some of our controller mains, since exiting immediately is actually what we want in that setting. But for CLI, it's better to return an error to display the help text, or be explicit about exiting without it, by calling `os.Exit`, which we do in a few places.

Reply:
Okay sounds good. Thank you for the explanation! I'll change these back to `return err`.